src/Core/Cache/MemcacheCacheDriver.php

```diff
@@ -35,6 +40,28 @@ public function __construct($memcache_host, $memcache_port)
 		}
 	}
 
+	/**
+	 * (@inheritdoc)
+	 */
+	public function getAllKeys()
+	{
+		$list = [];
+		$allSlabs = $this->memcache->getExtendedStats('slabs');
+		foreach($allSlabs as $slabs) {
```

Comment:
Just some standards here and in the following lines: Could you add a space after "foreach"?